docs/source/markdown/podman-pod-exists.1.md

```diff
@@ -14,20 +14,18 @@ was an issue accessing the local storage.
 
 ## EXAMPLES
 
-Check if a pod called `web` exists in local storage (the pod does actually exist).
+Check if a pod called `web` exists in local storage (the pod does actually exist):
 ```
 $ sudo podman pod exists web
 $ echo $?
 0
-$
 ```
 
-Check if a pod called `backend` exists in local storage (the pod does not actually exist).
+Check if a pod called `backend` exists in local storage (the pod does not actually exist):
```

Comment:
I find it a bit inconsistent that we're using an article for `pod` but not for `local storage`.  Shall we use "the local storage" in all cases?

Reply:
I think "local storage" is the generally-accepted term; like "system storage" (would sound weird with an article)